packages/shared/src/hooks/useBookmarkPost.ts

```diff
@@ -174,23 +174,19 @@ const useBookmarkPost = ({
       await addBookmark({ id: post.id });
       displayToast('Post was added to your bookmarks');
 
-      if (!seenBookmarkPromotion) {
-        const bookmarkLoops = getFeatureValue(feature.bookmarkLoops);
-
-        if (bookmarkLoops) {
-          completeAction(ActionType.BookmarkPromoteMobile);
-          openModal({
-            type: LazyModal.MarketingCta,
-            props: { marketingCta: promotion.bookmarkPromoteMobile },
-          });
-        }
+      if (!seenBookmarkPromotion && bookmarkPopup) {
```

Comment:
Should it enroll everyone or only is !seenBookmarkPromotion?

Reply:
Right. We can do that too.

Reply:
Ah, just realized, it wouldn't matter anyway since we load enroll this on the feed to show bookmark on card (yes we now have two flags for that, but this one includes popup).

Reply:
why did we have the need to change the way we get/use feature value here, seems the end result would be the same. This way we don't even doubt anything about enrolment anyway since we get the value after !seenBookmarkPromotion at least in this case even if it might not matter.

Reply:
Yeah makes sense since they already got enrolled :)

Reply:
@nensidosari we won't be using the old flag now. We should probably clean it up too.